src/app/components/accordion-group/accordion-group.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import $ from '~/helpers/$';
+import {
+    Accordion,
+    AccordionItem,
+    AccordionItemHeading,
+    AccordionItemButton,
+    AccordionItemPanel
+} from 'react-accessible-accordion';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faChevronUp} from '@fortawesome/free-solid-svg-icons/faChevronUp';
+import {faChevronDown} from '@fortawesome/free-solid-svg-icons/faChevronDown';
+import './accordion-group.scss';
+
+type FunctionOrFalse = false | ((n: unknown) => void);
+
+function useChevronDirection(
+    forwardOnChange: FunctionOrFalse,
+    preExpanded: unknown[]
+) {
+    const [openTabs, updateOpenTabs] = React.useState([...preExpanded]);
+    const chevronDirection = React.useCallback(
+        (uuid: string) => (openTabs.includes(uuid) ? 'up' : 'down'),
+        [openTabs]
+    );
+    const onChange = React.useCallback(
+        (newOpenTabs: unknown[]) => {
+            if (forwardOnChange) {
+                forwardOnChange(newOpenTabs);
+            }
+            updateOpenTabs(newOpenTabs);
+        },
+        [forwardOnChange, updateOpenTabs]
+    );
+
+    return [chevronDirection, onChange] as const;
+}
+
+type ChevronDirection = 'up' | 'down';
+
+function TitleBar({
+    title,
+    titleTag,
+    chevronDirection,
+    analytics
+}: {
+    title: string;
+    titleTag?: string;
+    chevronDirection: ChevronDirection;
+    analytics?: boolean;
+}) {
+    const icon = {
+        down: faChevronDown,
+        up: faChevronUp
+    }[chevronDirection];
+    const isOpen = chevronDirection === 'up';
+
+    return (
+        <AccordionItemHeading aria-level="2">
+            <AccordionItemButton
+                className="accordion-button"
+                data-analytics-link={analytics && !isOpen ? '' : undefined}
+            >
+                <div className="label">
+                    {title}
+                    {titleTag && <span className="title-tag">{titleTag}</span>}
+                </div>
```

Comment:
Double checking - we no longer want to render tooltipContent, or it wasn't ever used?